README.md

```diff
@@ -9,10 +9,10 @@
 
 Build | Status
 -- | --
-Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-fuel-tools/branch/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-fuel-tools/branch/main)

```

Comment:
We should keep `main` here, so this diff can be removed.

Reply:
Updated.  The buildfarm badges have also been updated to reflect `main`